### PR TITLE
First round of doc tests for column stats

### PR DESCRIPTION
**Pull request by ashahba**

### CODE REVIEW

File: doc-api-examples/src/main/resources/python/frame/column_median.rst [hunk 1]
```diff
@@ -1,5 +1,82 @@
...
+Compute and return middle number of values in column *a*:
+
+.. code::
+
+   <hide>
+   >>> median = my_frame.column_median('a')
+   <progress>
+   >>> print median
+   5
+   </hide>
```
Comment (blbarker) [inline doc-api-examples/src/main/resources/python/frame/column_median.rst]: why hide this from the docs?    (i.e. remove the <hide> tags)

File: doc-api-examples/src/main/resources/python/frame/column_median.rst [hunk 2]
```diff
@@ -1,5 +1,82 @@
...
+
+Given a frame with column 'a' and column 'w' as weights accessed by a Frame object 'my_frame':
+
+.. code::
+
+   <hide>
+   >>> import trustedanalytics as ta
```
Comment (blbarker) [inline doc-api-examples/src/main/resources/python/frame/column_median.rst]: Don't need to re-import or re-connect if in same file

File: doc-api-examples/src/main/resources/python/frame/column_median.rst [hunk 3]
```diff
@@ -1,5 +1,82 @@
...
+   [5]  10  0.800000011921
+   [6]  30   0.10000000149
+
+
+Compute and return middle number of values in column 'a' with weights 'w':
+
 .. code::
 
-    >>> median = frame.column_median('middling column')
+   <hide>
```
Comment (blbarker) [inline doc-api-examples/src/main/resources/python/frame/column_median.rst]: again, remove <hide>.  User needs to see this step --it's actually the most important step for the example :)

File: doc-api-examples/src/main/resources/python/frame/column_mode.rst [hunk 4]
```diff
@@ -1,5 +1,87 @@
...
+   [4]   7
+   [5]  10
+   [6]  30
+   
+
+Compute and return a dictionary containing summary statistics of column *a*:
+
+.. code::
+
+   <hide>
```
Comment (blbarker) [inline doc-api-examples/src/main/resources/python/frame/column_mode.rst]: again, remove <hide>

File: doc-api-examples/src/main/resources/python/frame/column_mode.rst [hunk 5]
```diff
@@ -1,5 +1,87 @@
...
+   >>> print  sorted(mode.values())
+   >>> [1, 2.0, 7.0, [3]]
+   </hide>
+
+Given a frame with column 'a' and column 'w' as weights accessed by a Frame object 'my_frame':
+
+.. code::
+
+   <hide>
+   >>> import trustedanalytics as ta
```
Comment (blbarker) [inline doc-api-examples/src/main/resources/python/frame/column_mode.rst]: again, redundant import and connect

File: doc-api-examples/src/main/resources/python/frame/column_mode.rst [hunk 6]
```diff
@@ -1,5 +1,87 @@
...
+   [5]  10  0.800000011921
+   [6]  30   0.10000000149
+   
+
+Compute and return dictionary containing summary statistics of column 'a' with weights 'w':
+
 .. code::
 
-    >>> mode = frame.column_mode('modum columpne')
+   <hide>
```
Comment (blbarker) [inline doc-api-examples/src/main/resources/python/frame/column_mode.rst]: remove <hide>

**Comment by ashahba** [comment]
Thanks Briton, I made your suggested changes and added summary statistics tests.
Please Let me know if you want more changes to be made before merge, otherwise it should be good for now.

**Pull request merged**